steely/plugins/snoop.py

```diff
@@ -16,7 +16,7 @@ def snoop(text):
     r = requests.post(GIZOOGLE_URL, data={'translatetext': text})
     html = r.text.replace('height:250px;"/>', 'height:250px;">')  # Don't ask
     soup = BeautifulSoup(html, 'html.parser')
-    return soup.textarea.string
+    return soup.textarea.string.encode('utf-8')
```

Comment:
`string#encode` turns the string into a bytes object. Are you sure this is what we want to do?